### PR TITLE
Add pvclust package for R

### PR DESCRIPTION
**Pull request by sebastian-luna-valero**

### CODE REVIEW

**Comment by sebastian-luna-valero** [comment]
Waiting for suggestions [here](https://github.com/conda-forge/staged-recipes/issues/3452)

**Comment by sebastian-luna-valero** [comment]
Please @conda-forge/staged-recipes review.

File: recipes/r-pvclust/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+  fn: pvclust_{{ version }}.tar.gz
+  url:
+    - https://cran.r-project.org/src/contrib/pvclust_{{ version }}.tar.gz
+    - https://cran.r-project.org/src/contrib/Archive/pvclust/pvclust_{{ version }}.tar.gz
+
+
+  sha256: 1a4615214992307fd7c786cf4607a3ae2af5c0d4067f5053e1c195798a70d741
+  # patches:
+   # List any patch files here
+   # - fix.patch
```
Comment (scopatz) [inline recipes/r-pvclust/meta.yaml]: Remove extra comment lines

**Comment by scopatz** [comment]
Thanks for putting in this PR!

**Comment by sebastian-luna-valero** [comment]
Close in favour of https://github.com/conda-forge/staged-recipes/pull/3872 ?

**Comment by sebastian-luna-valero** [comment]
Closing in favour of https://github.com/conda-forge/staged-recipes/pull/3872

**Pull request closed without merging**